TYPO3.Flow/Classes/TYPO3/Flow/Utility/Files.php

```diff
@@ -377,7 +378,11 @@ public static function unlink($pathAndFilename)
         try {
             // if not on Windows, call PHPs own unlink() function
             if (DIRECTORY_SEPARATOR === '/' || is_file($pathAndFilename)) {
```

Comment:
Another question out of curiosity:
Why do we use an `OR` operator and not `AND` here? I mean we want to proceed if where not on Windows AND when  the path leads to file, correct?